packages/discovery-react-components/src/components/DocumentPreview/components/ReactPdfViewer/typings.d.ts

```diff
@@ -0,0 +1,3 @@
+declare module 'react-pdf';
```

Comment:
You should be able to install `@types/react-pdf` and get rid of this file.

Reply:
Okay, will make this change when updating changes.